.gitignore

```diff
@@ -1,29 +1,31 @@
-.DS_Store
-node_modules
-/dist
-migrations
-
-# Python specific
-__pycache__/
-*.py[cod]
-
-# local env files
-.env.local
-*.env.*.local
-*.env
-
-# Log files
+# Logs
+logs
+*.log
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+pnpm-debug.log*
```

Comment:
We don't need stuff for

- [ ] `pnmp` or `lerna` (since we don't use either of those package management systems)
- [ ] `dist-ssr` (since we're not using server-side rendering)
- [ ] `coverage` (since we don't generate coverage reports)
- [ ] `*.local` (since our only `.env` file is just `.env` and if you have some sort of `.local` file that we don't instruct you to make, then it shouldn't be the responsibility of this `.gitignore` to ignore it)
- [ ] `/cypress/` stuff (since we don't use Cypress) 
- [ ] Editor stuff shouldn't be the under the purview of this `.gitignore`. (Ignore it globally or something.)

Reply:
Done in https://github.com/numberscope/frontscope/pull/123/commits/bc3b6cded6468d66d64ea9fecc5977e047d9d9e9.